feature-libs/cart/main/components/events/cart-components-event.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { AddToCartDialogEventListener } from './add-to-cart-dialog-event.listener';
+
+@NgModule({})
+export class CartComponentsEventModule {
+  constructor(_addToCartDialogEventListener: AddToCartDialogEventListener) {}
```

Comment:
can we move `add-to-cart-dialog-event.listener.ts` to `add-to-cart` folder, and remove this module? We can move this `constructor` to `AddToCartModule`.

Reply:
Done. Suggestion applied.